trusted_application_demo/optee_ta_security/src/Makefile

```diff
@@ -0,0 +1,20 @@
+CFG_TEE_TA_LOG_LEVEL = 2
+BINARY ?= aed3b02c-8e42-11e7-bb31be2e44b06b34
```

Comment:
I guess this needs to match https://github.com/ros2-for-arm/example/pull/4/files#diff-65f81c5dcada5b294cba5d34eaa5afe5R8 as well ? Maybe a comment here as well would be useful